schema.graphql

```diff
@@ -1,12 +1,63 @@
+scalar Date
+
 type Group {
   id: Int
   name: String
 }
 
+input GroupInput {
```

Comment:
Found a lot of examples online using the `input` type to declare query or mutation inputs that way they can be reused. For example, create and update mutations may often have the same params save for update needing an id:

```graphql
addShipment(input: ShipmentInput!): Shipment
updateShipment(id: Int!, input: ShipmentInput!): Shipment
```
```